secrets.json

```diff
@@ -0,0 +1,3 @@
+{
+    "mnemonic": "head pigeon debate ten curtain olive middle ride hat little hill bring"
+}
```

Comment:
Are you kidding? REMOVE THIS!
Do not share any private key or mnemonic to online, Hackers everywhere